CorsixTH/Lua/dialogs/message.lua

```diff
@@ -104,8 +104,9 @@ function UIMessage:adjustToggle()
   end
 end
 
+--! Displays the fax/strike message to the player when opened from the bottom_panel.
 function UIMessage:openMessage()
-  if TheApp.world:checkPauseBehaviour() and not self.ui:checkForMustPauses() then
+  if TheApp.world:checkPauseBehaviour() and not self.ui:checkForMustPauseWindows() then
```

Comment:
The `checkPauseBehaviour` function has a different name too.

Reply:
Looking at the wrong commit :)